manifests/v3.11.0/kube-apiserver/public-info-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
can you comment about the purpose? Maybe even put a comment into the yaml here.

Reply:
It's for the operator status object, right?

Is the role name agreed upon somewhere or is it our buest guess?